src/util/models/interactions.js

```diff
@@ -391,13 +391,19 @@ export class InteractionMessageEvent {
 
 		return responseMsg
 			.edit({
-				content: final,
+				content: final?.trim()?.length >= 2000 ? "" : final,
 				files: [
+					final?.trim()?.length >= 2000
+						? {
+								attachment: Buffer.from(final, "utf-8"),
```

Comment:
do we have to update this now cause of #69

Reply:
fuck.

Reply:
wait no, there's no support for long response links in passive activation yet (which is what this is about)

Reply:
oh ok